### DISCUSSION TITLE
100% API use vs Scraping?

### DISCUSSION THREAD

**Discussion by matt1966** [CyberPunkMetalHead/binance-trading-bot-new-coins#95] [Q&A]
Has anybody tested frequently hitting the Binance API for new coins as opposed to scraping their announcement page?

If so, what was the result of this?

**Comment by ADTC** [comment]
Announcements may come early. Or they may be late. They are manually written and posted by someone, quite possibly. The latest announcement for `THE` (Thena) came 30 minutes before its listing. The one before came 24 hours before its listing.

API is instantaneous, machine-friendly and programmatic. Just throttle yourself and avoid too many API calls.

I disabled the scraper for my use, to avoid slowing down the buy order with a useless dependency and unnecessary scraping.
